src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2935,6 +2935,14 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
                 throw new BadImageFormatException();
             }
 
+            // This blocks enforces the rule that methods with [NativeCallable] attribute
```

Comment:
Nit: typo ("this blocks enforces"), and a reference to `BadImageFormatException`, even though we changed this to `InvalidProgramException`.

Looks good otherwise!